api/test/round-tracker.test.js

```diff
@@ -826,8 +604,6 @@ describe('Round Tracker', () => {
 
         // Verify results
         const { rows: tasks } = await pgClient.query('SELECT miner_id, cid, clients, allocators FROM retrieval_tasks WHERE round_id = $1', [roundNumber])
-        assert.strictEqual(tasks.length, 1, 'Should have exactly one task')
-
```

Comment:
This seems suspicious. Was it intentional? Can we simply revert these to lines back?